physics/GFS_MP_generic.F90

```diff
@@ -42,12 +42,14 @@ subroutine GFS_MP_generic_pre_run(im, levs, ldiag3d, do_aw, ntcw, nncl, ntrac, g
         do k=1,levs
           do i=1,im
             save_t(i,k) = gt0(i,k)
-            save_q(1:im,:,1) = gq0(1:im,:,1)
```

Comment:
this assignment of the entire column inside the k loop didn't make any sense, thanks for fixing it

Reply:
There is an error here.  The save_qv variable should be used instead.